hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/InconsistentS3ClientFactory.java

```diff
@@ -45,7 +46,7 @@ public class InconsistentS3ClientFactory extends DefaultS3ClientFactory {
 
```

Comment:
i think we could just cut this class; it was there for generating inconsistencies more reliably. mockito is better for managed fault injection.

proposed; do it as a followup